src/Lorentz/Contracts/RegistryDAO.hs

```diff
@@ -0,0 +1,159 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+-- | A Registry DAO can be used to store key value pair @(k, v)@
+module Lorentz.Contracts.RegistryDAO
+  ( registryDaoContract
+  , config
+  )
+  where
+
+import Lorentz
+
+import qualified Lorentz.Contracts.BaseDAO as DAO
+import Lorentz.Contracts.RegistryDAO.Types
+import qualified Lorentz.Contracts.BaseDAO.Types as DAO
+import Universum hiding (drop, swap, (>>), some)
```

Comment:
AFAIU, `Lorentz` is supposed to be a self-contained prelude and it shouldn't be necessary to import `Universum` (or any other prelude) as well. There is [Lorentz.Prelude](https://hackage.haskell.org/package/lorentz-0.8.0/docs/Lorentz-Prelude.html) which exports a lot of prelude things. So I'd like to better understand what exactly we are importing from `Universum` here and whether it can be added to `Lorentz.Prelude`. And if it's 1-3 things, I think it's better to list them explicitly than listing 4 hidden functions.

Reply:
I see, I only import (*) which is use in voting period to make it more understandable: e.g. `DAO.cMaxVotingPeriod = 60 * 60 * 24 * 30 -- 1 months`.

I updated it to be `import Universum ((*))`